gettsim/parameters/elterngeld.yaml

```
@@ -110,7 +123,7 @@ elterngeld_geschwister_bonus_minimum:
   2007-01-01:
     scalar: 75
     reference: Art. 1 G. v. 05.12.2006 BGBl. I S. 2748
-elterngeld_mehrling_bonus:
+mehrling_bonus:
```

```suggestion
mehrlingbonus:
```
Nothing abbreviated here.